src/Api.Rest.Dtos.Tests/Benchmarks/SerializationBenchmark.cs

```diff
@@ -1,147 +1,187 @@
-﻿namespace Zeiss.PiWeb.Api.Rest.Dtos.Tests.Benchmarks
-{
-	#region usings
-
-	using System;
-	using System.Collections.Generic;
-	using System.Diagnostics.CodeAnalysis;
-	using BenchmarkDotNet.Attributes;
-	using BenchmarkDotNet.Configs;
-	using BenchmarkDotNet.Filters;
-	using BenchmarkDotNet.Running;
-	using Newtonsoft.Json;
```

Comment:
In this class only Newtonsoft Json is used. Why not rename the class to `NewtonsoftJsonSerializationBenchmark.cs`?

Reply:
No, both Newtonsoft.Json and System.Text.Json benchmarks are included for comparison.